### ISSUE TITLE
Usage of german umlaute

### ISSUE THREAD

**Issue by coolusaHD** [sinaatalay/RenderCV#29]
**Describe the bug**
When using characters like ÄÖÜ the generator returns
```
The input file contains a character that cannot be decoded with UTF-8 (b'\xdc'):
 b'LF/regular/n/10 \xdcber-set-zun-gen '
```

**To Reproduce**
Please provide a minimal YAML input as a code block for us to produce the same error.
```yaml
- start_date: 2023-10
        location: Berlin
        company: Test AG
        position: Werkstudent 
        highlights:
          - 'Umstellung der bestehende Übersetzungen auf eine neue Struktur, wodurch die Übersetzungen in der Software vereinheitlicht und die Wartbarkeit verbessert wurde'
```

**Comment by coolusaHD** [comment]
It looks like it has only a problem if the umlaute are at the beginning of a word

**Comment by sinaatalay** [comment]
Hello, I couldn't reproduce this error. Could you please check again to see if you are still having this issue?

**Comment by coolusaHD** [comment]
```
cv:
  name: Test
  location: test
  email: test@test.de
  phone: tel:+491748995625
  website: https://test.de
  social_networks:
    - network: LinkedIn
      username: test
    - network: GitHub
      username: test
  sections:
    ausbildung:
      - start_date: 2023-10
        location: Berlin
        company: Test AG
        position: Werkstudent
        highlights:
          - 'Umstellung der bestehende Übersetzungen auf eine neue Struktur, wodurch die Übersetzungen in der Software vereinheitlicht und die Wartbarkeit verbessert wurde'
design:
  theme: classic
  font_size: 10pt
  page_size: a4paper
  color: '#319da0'
  disable_page_numbering: false
  page_numbering_style: NAME - Seite PAGE_NUMBER von TOTAL_PAGES
  show_last_updated_date: false
  header_font_size: 30 pt
  text_alignment: justified
  margins:
    page:
      top: 2 cm
      bottom: 2 cm
      left: 2 cm
      right: 2 cm
    section_title:
      top: 0.3 cm
      bottom: 0.2 cm
    entry_area:
      left_and_right: 0.2 cm
      vertical_between: 0.2 cm
      date_and_location_width: 4.1 cm
    highlights_area:
      top: 0.10 cm
      left: 0.4 cm
      vertical_between_bullet_points: 0.10 cm
    header:
      vertical_between_name_and_connections: 0.3 cm
      bottom: 0.3 cm
      horizontal_between_connections: 0.5 cm
  show_timespan_in:
    - Experience

```

This produces:
```
PS C:\repos\cv> rendercv render .\test_german_umlaute.yaml

Welcome to RenderCV!
┏━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
┃ Title            ┃                                           Link ┃
┡━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┩
│ Documentation    │         https://sinaatalay.github.io/rendercv/ │
│ Source code      │        https://github.com/sinaatalay/rendercv/ │
│ Bug reports      │ https://github.com/sinaatalay/rendercv/issues/ │
│ Feature requests │ https://github.com/sinaatalay/rendercv/issues/ │
└──────────────────┴────────────────────────────────────────────────┘
╭─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
│ 0.0 s Reading and validating the input file has finished.                                                                                                           │
│ 0.0 s Generating the LaTeX file has finished.                                                                                                                       │
│ 0.0 s Generating the Markdown file has finished.                                                                                                                    │
│ 0.5 s Rendering the LaTeX file to a PDF has started.                                                                                                                │
╰─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
0.6 s ━━━━━━━━━━━━━━━━━━━━━━━━╺━━━━━━━━━━━━━━━ (3 out of 5 steps finished)

The input file contains a character that cannot be decoded with UTF-8 (b'\xdc'):
 b'LF/regular/n/10 \xdcber-set-zun-gen '
```

**Comment by coolusaHD** [comment]
@sinaatalay 
I've checked and it still occurs at version `1.5`

**Comment by sinaatalay** [comment]
Thank you. It helped me spot the problem. I will release the fix with v1.6

**Comment by coolusaHD** [comment]
Thank you very much
 @sinaatalay I tthink you can close this issue 
I will check with the new version and reopen if something still doesn't work
